RecoMuon/Configuration/python/RecoMuonPPonly_cff.py

```diff
@@ -49,22 +49,27 @@
 from Configuration.Eras.Modifier_fastSim_cff import fastSim
 
 # Muon Tracking sequence
-standalonemuontracking = cms.Sequence(standAloneMuonSeeds*standAloneMuons*refittedStandAloneMuons*displacedMuonSeeds*displacedStandAloneMuons)
+standalonemuontrackingTask = cms.Task(standAloneMuons,refittedStandAloneMuons,displacedMuonSeeds,displacedStandAloneMuons,standAloneMuonSeedsTask)
+standalonemuontracking = cms.Sequence(standalonemuontrackingTask)
 # not commisoned and not relevant in FastSim (?):
-fastSim.toReplaceWith(standalonemuontracking,standalonemuontracking.copyAndExclude([displacedMuonSeeds,displacedStandAloneMuons]))
-displacedGlobalMuonTracking = cms.Sequence(iterDisplcedTracking*displacedGlobalMuons)
-globalmuontracking = cms.Sequence(globalMuons*tevMuons*displacedGlobalMuonTracking)
+fastSim.toReplaceWith(standalonemuontrackingTask,standalonemuontrackingTask.copyAndExclude([displacedMuonSeeds,displacedStandAloneMuons]))
+displacedGlobalMuonTrackingTask = cms.Task(iterDisplcedTrackingTask,displacedGlobalMuons)
+displacedGlobalMuonTracking = cms.Sequence(displacedGlobalMuonTrackingTask)
+
+globalmuontrackingTask = cms.Task(globalMuons,tevMuons,displacedGlobalMuonTrackingTask)
+globalmuontracking = cms.Sequence(globalmuontrackingTask)
 # not commisoned and not relevant in FastSim (?):
 fastSim.toReplaceWith(globalmuontracking,globalmuontracking.copyAndExclude([displacedGlobalMuonTracking]))
```

Comment:
This is the problem. This should be changed to modify `globalmuontrackingTask` and it should exclude `displacedGlobalMuonTrackingTask`.